go/kbfs/libkbfs/prefetcher.go

```diff
@@ -699,7 +711,16 @@ func (p *blockPrefetcher) run(testSyncCh <-chan struct{}) {
 			ptr := ptrInt.(BlockPointer)
 			pre, ok := p.prefetches[ptr.ID]
 			if !ok {
-				p.log.Debug("nothing to cancel for block %s", ptr)
+				p.queuedPrefetchHandlesLock.Lock()
```

Comment:
Please do longish blocks like this in a helper function, so we can take advantage of `defer` for the unlock.